dhall-csv/LICENSE

```diff
@@ -0,0 +1,27 @@
+Copyright (c) 2020 Gabriel Gonzalez
```

Comment:
Not sure if this actually matters

```suggestion
Copyright (c) 2021 Gabriel Gonzalez
```

Reply:
It may be better to add dates rather than replace them, e.g. `2020-2021`.